src/components/Prescription/Prescription.css

```diff
@@ -1,12 +1,21 @@
 .script-container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-top: 22vh;
     border: black solid 3px;
     background-color: white;
-    height: 88%;
-    width: 95%; 
+    height: 70vh;
+    width: 40vw; 
```

Comment:
Here is where I refactored this to rely on its own size vs the container it is in.